congress/metadata/G000583.yaml

```diff
@@ -0,0 +1,2 @@
+name: Twitter
+link: https://pbs.twimg.com/profile_images/735547603912433664/TIW7-Jzs.jpg
```

Comment:
This image appears to be from the member's campaign. Two candidate replacements:

* Header from official page: https://gottheimer.house.gov/images/jg-headshot.png
* Avatar from https://twitter.com/RepJoshG